client/src/actions/newGrant.ts

```diff
@@ -20,7 +20,19 @@ export interface GrantCreated {
   owner?: string;
 }
 
-export type NewGrantActions = GrantCreated | NewGrantTXStatus;
+export const RESET_TX_STATUS = "RESET_TX_STATUS";
+export interface IPFSResetTXStatus {
```

Comment:
@schultztimothy is this field still in the IPFS state? I would move it to newGrant